exercises/practice/killer-sudoku-helper/src/test/java/KillerSudokuHelperTest.java

```diff
@@ -0,0 +1,57 @@
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class KillerSudokuHelperTest {
+
+    private KillerSudokuHelper helper = new KillerSudokuHelper();
+
+    @Test
+    public void testTrivialOneDigitCages() {
+        for (int n = 1; n <= 9; n++) {
+            List<List<Integer>> expected = List.of(List.of(n));
+            assertEquals(expected, helper.combinationsInCage(n, 1));
+        }
+    }
+
+    @Test
+    @Disabled("Remove to run test")
+    public void testCageWithSum45ContainsAllDigits() {
+        List<List<Integer>> expected = List.of(asList(1, 2, 3, 4, 5, 6, 7, 8, 9));
```

Comment:
Just me asking, but what is the difference between this and the following?

```java
List.of(List.of(1, 2, 3, 4, 5, 6, 7, 8, 9));
```

Reply:
`asList `creates mutable lists. I was trying something else out when I added that but finally didn't need it. Its better to use `List.of()` for consistency. Thanks.